### PR TITLE
chore: prepare for release 4.0.0

### PR DESCRIPTION
**Pull request by ozayr-zaviar**
## Summary
- Prepare for release 4.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Optimizely Python SDK Changelog
 
+## 3.11.0
+January 6th, 2022
+
+### New Features
+* Add a set of new APIs for overriding and managing user-level flag, experiment and delivery rule decisions. These methods can be used for QA and automated testing purposes. They are an extension of the OptimizelyUserContext interface ([#369](https://github.com/optimizely/python-sdk/pull/369), [#365](https://github.com/optimizely/python-sdk/pull/365), [#361](https://github.com/optimizely/python-sdk/pull/361)):
```
Comment (jaeopt) [inline CHANGELOG.md:7]: Can we change the orders to [361, 365, 369]?

**Review by jaeopt — CHANGES_REQUESTED** [review]
nit

**Review by jaeopt — APPROVED** [review]
LGTM

**Review by Mat001 — APPROVED** [review]
lgtm

**Pull request merged**